packages/features/webhooks/lib/scheduleTrigger.ts

```diff
@@ -451,3 +451,89 @@ export async function updateTriggerForExistingBookings(
   );
   await Promise.all(promise);
 }
+
+export async function listOOOEntries(
+  appApiKey?: ApiKey,
+  account?: {
+    id: number;
+    name: string | null;
+    isTeam: boolean;
+  } | null
+) {
+  const userId = appApiKey ? appApiKey.userId : account && !account.isTeam ? account.id : null;
+  const teamId = appApiKey ? appApiKey.teamId : account && account.isTeam ? account.id : null;
+
+  try {
+    const where: Prisma.OutOfOfficeEntryWhereInput = {};
+    if (teamId) {
+      where.user = {
+        teams: {
+          some: {
+            id: teamId,
+          },
+        },
+      };
+    } else if (userId) {
+      where.userId = userId;
+    }
+
+    // early return
+    if (!where.userId && !where.user) {
+      return [];
+    }
+
+    const oooEntries = await prisma.outOfOfficeEntry.findMany({
+      where: {
+        ...where,
+        // only get entries that are active and have not ended
+        // if the end date is in the past, the entry is considered inactive
+        end: {
+          gte: new Date(),
```

Comment:
why are we only returning active OOO entries here?

Reply:
Removing and now to return any 3 OOO entries found for that userId or teamId.
At first I thought Zapier will fetch from this but in the end it just uses it for obtaining types for example.